app/entry.client.tsx

```diff
@@ -5,6 +5,9 @@ import { hydrateRoot } from 'react-dom/client'
 if (ENV.MODE === 'development') {
 	import('~/utils/devtools.tsx').then(({ init }) => init())
 }
+if (ENV.MODE === 'production') { 
```

Comment:
I wonder if we should add the sentry dsn env var to this check so we can have this bundle not load if they've not setup sentry yet. Would that do any harm?

Reply:
@kentcdodds No harm, I can make that change.